com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -772,7 +772,7 @@ void DisposeProbeCameraPool()
 #endif
             }
 
-            if (IsAPVEnabled())
+            if (IsAPVEnabled() && ProbeReferenceVolume.instance.isInitialized)
```

Comment:
Shouldn't we be able to test isInitialized inside the cleanup function and avoid having this public?
If we do it this way it means that any SRP needs to replicate this not too obvious pattern

Reply:
I did think about that, but the problem is that if cleaning up happens _anywhere else_ but this specific use case, then if the system is not initialized something is probably wrong. 

Now, one could argue in that case we could just not cleanup, but I wondered if that "hides" potential issues.